src/draw-svg.ts

```diff
@@ -106,6 +108,67 @@ function drawArcArrows(x: Arc, type: LineArrow) {
   return path;
 }
 
+/** Returns the four Cubic Bezier points need to round off a corner */
+export function getBezierPoints(points: Point[], radius: number) {
+  const length0 = Point.distance(points[0], points[1]);
+  const length1 = Point.distance(points[1], points[2]);
+
+  const r1 = Math.max(0.1, length0/2);
+  const r2 = Math.max(0.1, length1/2);
+  const rad = Math.min(radius, r1, r2);
+
+  const d1 = rad/length0;
+  const d2 = rad/length1;
+  const shift = 1 - CIRCLE_MAGIC;
+
+  const p1 = Point.interpolate(points[0], points[1], clamp(1 - d1, 0, 1));
+  const p2 = Point.interpolate(points[0], points[1], clamp(1 - d1*shift, 0, 1));
+  const p3 = Point.interpolate(points[1], points[2], clamp(d2*shift, 0, 1));
+  const p4 = Point.interpolate(points[1], points[2], clamp(d2, 0, 1));
+  const offsets = [p1, p2, p3, p4];
```

Comment:
Nit: You can just return the array directly, rather than assigning it to a variable